lib/server.js

```diff
@@ -47,6 +47,19 @@ module.exports = function createServer(config, log) {
 
   var handleBan = P.promisify(require('./bans/handler')(config.memcache.recordLifetimeSeconds, mc, EmailRecord, IpRecord, log))
 
+  // Setup blocklist
+  if (config.ipBlocklist.enable) {
+    var IPBlocklist = require('./ip_blocklist')(log)
+    var ipBlocklist = new IPBlocklist()
+    ipBlocklist.load(config.ipBlocklist.listPath)
```

Comment:
It seems unlikely, but could this load hang longer than expected, and so no be ready when requests are serving?